File: common/pci_io.c
/*
 * Cisco router simulation platform.
 * Copyright (c) 2006 Christophe Fillot (cf@utc.fr)
 *
 * PCI I/O space.
 */

#include "rust_dynamips_c.h"

#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <unistd.h>
#include <sys/types.h>

#include "cpu.h"
#include "vm.h"
#include "dynamips.h"
#include "pci_io.h"

File: dynamips-c/src/pci_io.rs
//! Cisco router simulation platform.
//! Copyright (c) 2005,2006 Christophe Fillot (cf@utc.fr)

use crate::device::*;
use crate::pci_dev::*;

// PCI I/O data
#[repr(C)]
#[derive(Debug, Copy, Clone)]
pub struct pci_io_data {
    pub dev: vdevice,
    pub dev_list: *mut pci_io_device,
}
